src/components/StudyHeader/StudyHeader.js

```diff
@@ -25,7 +50,23 @@ const StudyHeader = ({
 
   return (
     <Container>
-      <Header as="h1">{studyName || 'Unknown study name'}</Header>
+      {newStudy && newStudy.length > 0 && (
+        <small
+          className={statusyMessage[currentStatus].class}
+          onClick={() => {
+            showModal(true);
+          }}
+        >
+          <Icon name={statusyMessage[currentStatus].icon} />
+          {statusyMessage[currentStatus].text}
+        </small>
+      )}
+      <Header
+        as="h1"
+        className={newStudy && newStudy.length > 0 ? 'mt-15' : ''}
```

Comment:
reverse this so that the `mt-15` is added only if there is **NOT** a new study.

Reply:
'mt-15' here is actually making the margin smaller, by default there is a bigger top margin from Header component.